### PR TITLE
feat: update and adjust streamr integration

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

File: tools/streamr/create-stream.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  STREAMR_STORAGE_NODE_GERMANY,
+} from "streamr-client";
+
+const privateKey = "";
+const streamPath = "/redstone-oracle/";
+
+(async () => {
+  try {
+    const streamrClient = new StreamrClient({
+      auth: { privateKey: privateKey },
```
Comment (hatskier) [inline tools/streamr/create-stream.ts:13]: Can you please replace `{ privateKey: privateKey }` with `{ privateKey }`?

File: tools/streamr/create-stream.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import {
+  StreamrClient,
+  StreamPermission,
+  STREAMR_STORAGE_NODE_GERMANY,
+} from "streamr-client";
+
+const privateKey = "";
```
Comment (hatskier) [inline tools/streamr/create-stream.ts:7]: Can we ask for private key using prompts module?

File: tools/streamr/create-stream.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import {
+  StreamrClient,
+  StreamPermission,
+  STREAMR_STORAGE_NODE_GERMANY,
+} from "streamr-client";
```
Comment (hatskier) [inline tools/streamr/create-stream.ts:5]: Can you add some comments to this file to tell what are the main requirements for creating a new stream (e.g. minimal required Matic balance)?

**Review by hatskier — CHANGES_REQUESTED** [review]

**Comment by cehali** [comment]
Updated on monorepo

**Pull request closed without merging**